### ISSUE TITLE
URI-Encoding not reverted when filling variable / not compatible with paper-textbox

### ISSUE THREAD

**Issue by Compufreak345** [atotic/excess-router#4]
Hi,

I'm really glad I found your router - but there seems to be an issue with URI-encoding. I am using the router to prefill a login-field and as soon as I write an "@", it gets escaped - not only in the URI, but also in the textbox. Tested in Firefox & Chrome on OS X 10.11

You can reproduce the issue here : 

http://output.jsbin.com/cabuyo#/test 

Try writing an "@" in the textbox and you see what I mean. It also seems to enter some kind of escape-mode so I can't even use backspace afterwards.

Source : 
http://jsbin.com/cabuyo/edit?html,output

(Example only working in fullscreen mode) 

Best regards,
Christoph

/edit: I opened a [pull request](https://github.com/atotic/excess-router/pull/5) that should fix the issue. I hope I found the correct place ;)

You can try out the fixed version here : 
http://output.jsbin.com/wufadi#/test

**Comment by Compufreak345** [comment]
Sorry for closing, I wasn't aware that merging it in my fork closes the issue in your project :0

**Comment by atotic** [comment]
you've opened up a rats nest with the uri encodings, I've been routing happily with my non special-character routes. Working on it, it might take a bit of time to cover all the special cases.

**Comment by atotic** [comment]
Turns out fix was a one-liner. Try out master, tell me if it worked.
PS: saw you starred fancy-doc-viewer, I've abandoned it, feel free to continue.

**Comment by Compufreak345** [comment]
Well, I can use the input box without issues again - that's good. Unfortunately if I fill the box, than copy the URI and paste it into another window, the characters are not unescaped. For my use case this doesn't matter right now, but it probably also needs a fix.

P.S.: Yeah I saw that fancy-doc-viewer was abandoned, I starred it to remember in the future and maybe port it to Polymer 1.0 - but I'm not sure if I will find the time for it (or find another documentation solution). It looks quite useful.
